aci/resource_aci_bgpprotp.go

```diff
@@ -274,17 +274,17 @@ func resourceAciL3outBGPProtocolProfileRead(ctx context.Context, d *schema.Resou
 	bgpRsBestPathCtrlPolData, err := aciClient.ReadRelationbgpRsBestPathCtrlPol(dn)
 	if err != nil {
 		log.Printf("[DEBUG] Error while reading relation bgpRsBestPathCtrlPol %v", err)
-		d.Set("relation_bgp_rs_best_path_ctrl_pol", "")
+		setRelationAttribute(d, "relation_bgp_rs_best_path_ctrl_pol", "")
```

Comment:
why we don't use this setRelationAttribute() in Import func?

Reply:
We are not importing relationships in any of the resources. We need to update all resources if we decide to import relationships.

Reply:
what's the difference between setRelationAttribute and d.Set() there?